proto/babylon/mint/v1/mint.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package babylon.mint.v1;
+
+option go_package = "github.com/babylonlabs-io/babylon/x/mint/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/timestamp.proto";
+
+// Minter represents the mint state.
+message Minter {
```

Comment:
we don't need params?

Reply:
I think having params in the inflation protocol is deliberately avoided because we don't want gov prop that updates the inflation, at least in the very beginning of the protocol. Apart from typical reasons, one more reason for Babylon is that we are not fully sure whether BTC stakers can get voting power in governance or not.

For now wdyt about adding a TODO there?

Reply:
sounds good to me